docs/releases/overview.md

```diff
@@ -41,6 +41,54 @@ The primary focus of this release has been to add general bug fixes and improvem
+
+#### Refactor datanode API for getting balance history
+The API field `GetBalanceHistory` has been renamed to `ListBalanceHistory` and has had improvements in the documentation to help users understand APIs the 'grouping' feature. This change aslo fixes an issue with leaking account IDs. This work was done under the issue [6513 ↗](https://github.com/vegaprotocol/vega/issues/6513).
+
+#### Critical Bug fixes
+
+#### Price monitoring price-range cache restored incorrectly
+When restoring the `pricemontior` for a market from a snapshot, the integer-representation from the `wrappedDecimcal`'s used for the price-range cache are derived from the decimal representation. This is slightly different to how they are created in the normal, non-snapshot code path. This causes markets to act differently after a snapshot restore, and eventually the node falls out of consensus. This fix was implemented under issue [6525](https://github.com/vegaprotocol/vega/issues/6525).
+
+#### New features: Core
+
+#### Add reason to stopped or rejected transfer events
+In order to know why a transfer event has been stopped or rejected the transfer rejection error reason has now been exposed in `BUS_EVENT_TYPE_TRANSFER` events. This work was done under the issue [6529 ↗](https://github.com/vegaprotocol/vega/issues/6529)
+
+#### Update Tendermint to v0.34.22
+To keep Tendermint up-to-date with all of the latest bug fixes it has been upgraded to v0.34.22. To find out more about the changes please see the [Tendermint changelog](https://github.com/tendermint/tendermint/blob/v0.34.22/CHANGELOG.md#v03422). This work was done under the issue [6548 ↗](https://github.com/vegaprotocol/vega/issues/6548).
+
+#### New features: Datanode
+
+#### Data-node handles upgrade block and ensures data is persisted before upgrade
+In order to ensure that the whole state of the data-node is matching that of the validator nodes, the data-node should ensure that it processes all blocks up to the block height of a scheduled upgrade before shutting down. Respectively the core node shouldn't shutdown until the data-node has consumed all the blocks in the broker queue. This work was done under the issue [6080 ↗](https://github.com/vegaprotocol/vega/issues/6080).
+
+#### Add last-block sub-command to datanode CLI
+To make the Vega Visor UX easier to restart a node on the network, a command has been added to the datanode that will return the height of the last block committed, this will make it easier for Visor to figure at what snapshot height it should start the core. This work was done under the issue [6527 ↗](https://github.com/vegaprotocol/vega/issues/6527).
```

Comment:
```suggestion
#### Add last-block sub-command to data node CLI
To make the Vega Visor UX easier to restart a node on the network, a command has been added to the data node software that will return the height of the last block committed. This will make it easier for Visor to know at what snapshot height it should start the core. This work was done under the issue [6527 ↗](https://github.com/vegaprotocol/vega/issues/6527).
```